src/android/ContentFilesystem.java

```diff
@@ -43,6 +45,15 @@ public ContentFilesystem(Context context, CordovaResourceApi resourceApi) {
 
     @Override
     public Uri toNativeUri(LocalFilesystemURL inputURL) {
+        String path = inputURL.uri.getPath();
+        if (path != null && path.contains(PHOTO_PICKER_PREFIX)) {
+            // This is a Photo Picker URI, convert it to a content:// URI
+            String location = path.substring(
+                    path.lastIndexOf(PHOTO_PICKER_PREFIX) + PHOTO_PICKER_PREFIX.length(),
+                    path.lastIndexOf('.'));
```

Comment:
I'm not getting the `path.lastIndexOf('.')` part. Why are we removing the file's extension?

Reply:
The android content URIs don't have the file extensions
for example: `content://media/picker/0/com.android.providers.media.photopicker/media/1000000198`